ECMA2Yaml/ECMAHelper/Models/SDP/ItemSDPModelBase.cs

```diff
@@ -103,6 +103,46 @@ public ItemSDPModelBase()
         [YamlMember(Alias = "examples")]
         public string Examples { get; set; }
 
+        [JsonProperty("requirement_sdk_names")]
```

Comment:
can we better organize these properties into a dedicated object for better management? For example:
- uwpProperties
  - SDKRequirements
    - SDPRequirement
      - name
      - url
  - OSRequirements
    - OSRequirement
      - name
      - minVersion